src/pages/Onboard/NameTeam.vue

```diff
@@ -130,18 +136,92 @@ export default {
       await this.createLicense()
       if (!this.updateServerError) this.goToResources()
     },
-    goToResources() {
+    async accept(pt) {
+      try {
+        this.accepting = true
+        const invitationId = pt.id
+        const accepted = await this.acceptMembershipInvitation(invitationId)
+        this.redirectTenant = pt.tenant.slug
+        if (accepted.accept_membership_invitation.id) {
+          this.redirectTenant = pt.tenant.slug
+          this.accepting = false
+        }
+      } catch (e) {
+        this.mutationErrorMessage = e
+          .toString()
+          .split(':')
+          .pop()
+        this.loading = false
+        this.error = true
+      }
+      this.loading = false
+    },
+    async decline(id) {
+      try {
+        this.deleting = true
+        const declined = await this.declineMembershipInvitation(id)
+        if (declined.delete_membership_invitation.success) {
+          this.$emit('hide')
+        }
+        this.deleting = false
+      } catch (e) {
+        this.mutationErrorMessage = e
+          .toString()
+          .split(':')
+          .pop()
+        this.loading = false
+        this.error = true
+      }
+    },
+    async goToResources() {
       this.revealNameInput = false
       this.revealUrlInput = false
       this.revealConfirm = false
 
+      await this.setCurrentTenant(
+        this.redirectTenant != null
+          ? this.redirectTenant
+          : this.tenantChanges.slug || this.tenant.slug
```

Comment:
```suggestion
        this.redirectTenant ?? this.tenantChanges.slug ?? this.tenant.slug
```
We could probably coalesce these to make it clearer than the ternary.